cmd/kubermatic-installer/cmd_deploy.go

```diff
@@ -213,7 +229,8 @@ func DeployFunc(logger *logrus.Logger, versions kubermatic.Versions, opt *Deploy
 			MLASkipLogging:                     opt.MLASkipLogging,
 			Versions:                           versions,
 			SkipCharts:                         opt.SkipCharts,
-			DeployDefaultAppCatalog:            opt.DeployDefaultAppCatalog,
+			DeployDefaultAppCatalog:            defaultAppCatalog.Enabled,
+			LimitApps:                          defaultAppCatalog.LimitApps,
```

Comment:
Why? You'd already have the entire KubermaticConfiguration here, why have another field for this?